ceph_bridge.spec

```diff
@@ -0,0 +1,68 @@
+%define pkg_name tendrl-ceph-bridge
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
+BuildArch: noarch
+Summary: Tendrl bridge for Ceph Storage
+Source0: %{pkg_name}-%{pkg_version}.tar.gz
+Group:   Applications/System
+License: GPLv2+
+Url: https://github.com/Tendrl/ceph_bridge
+
+Requires: python-etcd >= 0.4.3
+Requires: python-dateutil >= 2.4.2
+Requires: python-gevent >= 1.0.2
+Requires: python-greenlet >= 0.4.2
+Requires: pytz
+Requires: python-msgpack >= 0.4.5
+Requires: python-taskflow >= 2.6.0
+Requires: tendrl-bridge-common
+
+%description
+Tendrl bridge for Ceph Storage
+
+%prep
+%setup -n %{pkg_name}-%{pkg_version}
+# Remove the requirements file to avoid adding into
+# distutils requiers_dist config
+rm -rf {test-,}requirements.txt
+
+# Remove bundled egg-info
+rm -rf %{pkg_name}.egg-info
```

Comment:
Sure, i will remove in tar also.
Keeping it here will help if someone uses rpmbuild command.